smarts/core/agent_interface.py

```diff
@@ -102,6 +102,13 @@ class Accelerometer:
     pass
 
 
+@dataclass
+class Goalpoints:
+    """ Observation of nearby goalpoints """
+
+    near_acquisition_radius: float = 50
```

Comment:
Can you just call it radius? Goal radius seems pretty clear what it is.

Reply:
The intention here is sensor radius, I have clarified it.